index.html

```diff
@@ -303,6 +303,79 @@ <h4>+</h4>
           </div>
       </div>
     </div>
+        
+    <div class="sponsors" style="background-color: grey;">
+      <div class="container" style="padding: 3%; padding-bottom: 10%">
+          <div class="row">
+            <div class="col-sm">
+                <div class="sponsors-header">
+                <h1>Sponsors</h1>
+                </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="sponsors-list" style="background-color: grey;">
+                <img class="sponsor-img" src="./img/w_facebook.svg" alt="">
+                <img class="sponsor-img" src="./img/w_instagram.svg" alt="">
+                <img class="sponsor-img" src="./img/w_linkedin.svg" alt="">
+                <img class="sponsor-img" src="./img/w_medium.svg" alt="">
+                <img class="sponsor-img" src="./img/w_twitter.svg" alt="">
+                <img class="sponsor-img" src="./img/w_youtube.svg" alt="">
+            </div>
+          </div>
+      </div>
+    </div>
+        
+    <div class="contact">
+      <div class="container" style="padding: 1%;">
+        <div class="contact-header">
+          <h4>Contact Us</h4>
+        </div>
+        <div class="contact-content">
+            <p>Reach out to us at <a href="mailto:hello@gotechnica.org">hello@gotechnica.org</a></p>
+        </div>
+        <div class="social-buttons">
+            <div class="social-circle1">
```

Comment:
su'ad actually already made classes for these social buttons! lines 85-100 in the .scss file where she has "blue-circle" "purple-circle" etc. lets use those instead (i think the size she had for those is better too)